docs/concepts/scheduled-jobs.md

```diff
@@ -0,0 +1,37 @@
+(scheduled-jobs)=
+# Scheduled Jobs
+
+Often times you may want to repeadedly run a `Job` on a regular schedule. For example, fetching from a datasource every morning, compiling an analytics report every month, or detecting model drift every hour.
+
+### Create a Job
+
+MLRun makes it very simple to add a schedule to a given `Job`. To showcase this, we will add a schedule to run the following `schedule.py` file every hour:
```

Comment:
MLRun makes it very simple to add a schedule to a given `job`. First, create the `job` including the `schedule.py` file: